crates/rpc/rpc/src/eth/api/call.rs

```diff
@@ -54,6 +54,7 @@ where
         request: CallRequest,
         block_number: Option<BlockId>,
         state_overrides: Option<StateOverride>,
+        _block_overrides: Option<Box<BlockOverrides>>,
```

Comment:
@mattsse Thanks for your mentoring :) Indeed, I saw that the EvmOverrides structure encapsulates BlockOverrides directly. I thought adding the block_overrides argument provided direct access but I was wrong :)

In the case of using EvmOverrides, if I'm not mistaken, on this call the block_overrides argument is unused because transact_call_at doesn't need it (it has already EvmOverrides type in it) correct?